packages/core/components/FileList/useFileAccessContextMenu.ts

```diff
@@ -8,6 +8,138 @@ import FileFilter from "../../entity/FileFilter";
 import { interaction, selection } from "../../state";
 import { AnnotationName } from "../../entity/Annotation";
 
+export function useFileAccessMenuItems(
+    plateLink?: string,
+    filePath?: string,
+    filters?: FileFilter[]
+): IContextualMenuItem[] {
+    const dispatch = useDispatch();
+    const isOnWeb = useSelector(interaction.selectors.isOnWeb);
+    const fileSelection = useSelector(selection.selectors.getFileSelection);
+    const isQueryingAicsFms = useSelector(selection.selectors.isQueryingAicsFms);
+    const userSelectedApplications = useSelector(interaction.selectors.getUserSelectedApplications);
+    const { executionEnvService } = useSelector(interaction.selectors.getPlatformDependentServices);
+
+    return React.useMemo(() => {
+        const savedApps: IContextualMenuItem[] = (userSelectedApplications || []).map((app) => {
+            const name = executionEnvService.getFilename(app.filePath);
+            return {
+                key: `open-with-${name}`,
+                text: name,
+                title: `Open files with ${name}`,
+                onClick() {
+                    dispatch(interaction.actions.openWith(app, filters));
+                },
+            };
+        });
+
+        savedApps.push({
+            key: ContextMenuActions.OPEN_3D_WEB_VIEWER,
+            text: "3D Web Viewer",
+            title: `Open files with 3D Web Viewer`,
+            href: `https://allen-cell-animated.github.io/website-3d-cell-viewer/?url=${filePath}/`,
+            disabled: !filePath,
+            target: "_blank",
+        });
+        if (plateLink) {
+            savedApps.push({
+                key: ContextMenuActions.OPEN_PLATE_UI,
+                text: "LabKey Plate UI",
+                title: "Open this plate in the Plate UI",
+                href: plateLink,
+                target: "_blank",
```

Comment:
LabKey open to a page that says `Hello from production loadbalancer.` Is the intended page?

Reply:
Huh yeah it seems to be loading the correct page, I didn't change the URL this directs to (intentionally anyway). I'll add an issue to follow this up with Infra to see if we need to use a different URL or if this loadbalancer is broken